client/my-sites/stats/stats-subscribers-chart-section/subscribers-navigation-arrows.tsx

```diff
@@ -0,0 +1,77 @@
+import NavigationArrows from '../navigation-arrows';
+
+type SubscribersNavigationProps = {
+	date: Date;
+	period: keyof typeof periodFunctions;
+	quantity: number;
+	onDateChange: ( newDate: Date ) => void;
+};
+
+const isPast = ( date: Date ): boolean => {
+	const incomingNextDate = new Date( date );
+	incomingNextDate.setDate( incomingNextDate.getDate() + 1 );
+	incomingNextDate.setHours( 0, 0, 0, 0 );
+
+	const today = new Date();
+	today.setHours( 0, 0, 0, 0 );
+
+	return incomingNextDate.getTime() < today.getTime();
+};
+
+const periodFunctions = {
+	day: ( date: Date, quantity: number ) => date.setDate( date.getDate() + quantity ),
+	week: ( date: Date, quantity: number ) => date.setDate( date.getDate() + quantity * 7 ),
+	month: ( date: Date, quantity: number ) => date.setMonth( date.getMonth() + quantity ),
+	year: ( date: Date, quantity: number ) => date.setFullYear( date.getFullYear() + quantity ),
+};
```

Comment:
How could we make this function map easier to understand? Maybe the following might help:

- Renaming the map to `addQuantityToDate`.
- Increasing parameter specificity, like `( date: Date, daysToBeAdded: number )`.